server/datastore/mysql/activities_test.go

```diff
@@ -453,3 +480,27 @@ func testListHostUpcomingActivities(t *testing.T, ds *Datastore) {
 		})
 	}
 }
+
+func testListHostPastActivities(t *testing.T, ds *Datastore) {
+	ctx := context.Background()
+
+	u := test.NewUser(t, ds, "user1", "user1@example.com", false)
+
+	h1 := test.NewHost(t, ds, "h1.local", "10.10.10.1", "1", "1", time.Now())
+	opt := fleet.ListActivitiesOptions{
+		ListOptions: fleet.ListOptions{
+			Page:    0,
+			PerPage: 1,
+		},
+	}
+
+	require.NoError(t, ds.NewActivity(context.Background(), u, fakeHostActivity{
+		name:    "run script 1",
+		details: map[string]interface{}{"detail": 1, "sometext": "aaa"},
+		hostIDs: []uint{1},
+	}))
+
+	acts, _, err := ds.ListHostPastActivities(ctx, h1.ID, opt.ListOptions)
```

Comment:
Probably worth testing the pagination and ordering a bit, to make sure the dynamically-built SQL query is working as expected.

Reply:
Yeah, I beefed this test up quite a bit.